scripts/restore.py

```diff
@@ -53,7 +53,7 @@ def run_command(cmd):
 count = psql_output.split("\n")[2].strip()
 if count > 2:
     logger.info("Too many postgres threads(" + str(count) + ")... restarting")
-    run_command("service {}-postgresql restart".format(db.postgres_version()))
+    run_command("service {}-postgresql restart".format(current_appliance.postgres_version))
```

Comment:
afaik we are trying to get rid of current_appliance but unfortunately, I don't know what has to be used in this case instead. @psav, could you advice ?